docs/csharp/toc.yml

```diff
@@ -62,9 +62,9 @@
     href: tutorials/console-webapiclient.md
   - name: Inheritance in C# and .NET
     href: tutorials/inheritance.md
-  - name: Working with LINQ
+  - name: Work with Language-Integrated Query (LINQ)
```

Comment:
Do we need to spell out acronyms on the TOC too? It makes it a bit more verbose. I'd ensure that the article has that expanded (which right now it doesn't)

Reply:
I'm not sure - I changed it back to just the acronym and added it to the article.